dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentsExportController.java

```diff
@@ -147,13 +147,14 @@ public PagingWrapper<ObjectNode> getEnrollments( RequestParams requestParams )
     @OpenApi.Response( OpenApi.EntityType.class )
     @GetMapping( value = "/{uid}" )
     public ResponseEntity<ObjectNode> getEnrollmentByUid(
-        @OpenApi.Param( { UID.class, Enrollment.class } ) @PathVariable String uid,
```

Comment:
I think you should only remove `UID.class` once you make changes to the `OpenApi` spec generation. I assume the `OpenAPI` spec is not as detailed as before when removing `UID.class`.

Reply:
Make sense. I will put it back and we can improve that part later.